core/domain/story_domain.py

```diff
@@ -48,6 +49,8 @@
 # These take node_id as parameter.
 CMD_ADD_STORY_NODE = 'add_story_node'
 CMD_DELETE_STORY_NODE = 'delete_story_node'
+CMD_FINALIZE_STORY_NODE_OUTLINE = 'finalize_story_node_outline'
+CMD_RESET_STORY_NODE_OUTLINE_STATUS = 'reset_story_node_outline_status'
```

Comment:
Maybe collapse these into CMD_UPDATE_STORY_NODE_OUTLINE_STATUS and introduce an additional self.finalized boolean.

("Reset" doesn't automatically imply "unfinalize" IMO.)

Reply:
Done.